src/Helper/OrganisationApiHelper.php

```diff
@@ -26,167 +30,166 @@ class OrganisationApiHelper {
-
-      return $responseArray['hydra:member'] ?? [];
-    }
-    else {
+    if (Response::HTTP_OK != $response->getStatusCode()) {
       return [];
     }
+
+    $responseArray = $this->getResponseContentsAsArray($response);
+
+    return $responseArray['hydra:member'] ?? [];
   }
 
   /**
    * Get manager information for bruger.
    *
    * @phpstan-return array<string, mixed>
+   *
+   * @throws \Drupal\os2forms_organisation\Exception\ApiException
+   *   API exception.
    */
   public function getManagerInformation(string $brugerId): array {
-    $client = $this->getCurlClient();
-
-    $response = $client->request('GET', $this->settings->getOrganisationApiEndpoint() . 'bruger/' . $brugerId . '/leder');
+    $response = $this->get('bruger/' . $brugerId . '/leder');
 
-    if (200 === $response->getStatusCode()) {
-
-      $managerIds = $response->toArray()['hydra:member'];
+    if (Response::HTTP_OK != $response->getStatusCode()) {
+      return [];
+    }
 
-      // Select first manager if more than one.
-      if (count($managerIds) >= 1) {
-        $managerId = reset($managerIds);
-      }
-      else {
-        return [];
-      }
+    $managerIds = $this->getResponseContentsAsArray($response)['hydra:member'] ?? [];
 
-      return $this->getBrugerInformationer($managerId);
+    // Select first manager if more than one.
+    if (count($managerIds) >= 1) {
+      $managerId = reset($managerIds);
     }
     else {
       return [];
     }
+
+    return $this->getBrugerInformationer($managerId);
   }
 
   /**
    * Get (first) manager id for bruger.
+   *
+   * @throws \Drupal\os2forms_organisation\Exception\ApiException
+   *   API exception.
    */
   public function getManagerId(string $brugerId): string {
-    $client = $this->getCurlClient();
-
-    $response = $client->request('GET', $this->settings->getOrganisationApiEndpoint() . 'bruger/' . $brugerId . '/leder');
+    $response = $this->get('bruger/' . $brugerId . '/leder');
 
-    if (200 === $response->getStatusCode()) {
-
-      $managerIds = $response->toArray()['hydra:member'];
+    if (Response::HTTP_OK != $response->getStatusCode()) {
+      return '';
+    }
 
-      // Select first manager if more than one.
-      if (count($managerIds) >= 1) {
-        return reset($managerIds);
-      }
-      else {
-        return '';
-      }
+    $managerIds = $this->getResponseContentsAsArray($response)['hydra:member'] ?? [];
 
+    // Select first manager if more than one.
+    if (count($managerIds) >= 1) {
+      return reset($managerIds);
     }
-    else {
-      return '';
-    }
+
+    return '';
   }
 
   /**
    * Search for bruger.
    *
    * @phpstan-return array<string, mixed>
+   *
+   * @throws \Drupal\os2forms_organisation\Exception\ApiException
+   *   API exception.
    */
   public function searchBruger(string $query): array {
-    $client = $this->getCurlClient();
+    $response = $this->get('bruger?page=1&navn=' . $query);
```

Comment:
You should not manually build the query string. Pass an array with the parameters to the `get` function:

```suggestion
    $response = $this->get('bruger', ['page' => 1, 'navn' => $query]);
```

Reply:
Used `get` to build query string.